packages/browser/README.md

```diff
@@ -0,0 +1,170 @@
+# Backtrace Browser
+Backtrace error reporting tool for browser side JavaScript
+ 
+## Table of Contents
+1. [Integration Example](#integration-example)
+1. [Features](#features)
+    - [File Attachments](#file-attachments)
+    - [Metrics](#metrics)
+    - [Breadcrumbs](#breadcrumbs)
+    - [Callback Methods](#callback-methods)
+1. [Documentation](#documentation)
+
```

Comment:
We also need to prompt the user to integrate source maps and source code...

Reply:
Done.